public/js/register.js

```diff
@@ -15,8 +15,8 @@ function clearSignUpForm(){
     let first_name = document.getElementById('first_name');
     let last_name = document.getElementById('last_name');
     let email = document.getElementById('email');
-    let username = document.getElementById('username');
-    let password = document.getElementById('password');
+    let username = document.getElementById('user');
+    let password = document.getElementById('pass');
```

Comment:
This may just be me nit-picking, but I am a little concerned about the approach to call the same things (`username` and `password`) different names just as a workaround. Can we figure out a way to not have the two conflicting HTML forms on the same page? For example, when user clicks on register on the login page, then just take them to a dedicated `/register` page.

Reply:
I don't think we have a dedicated register page, which is why that was the solution I went with. It seems like the register feature was implemented as a pop up window. I think it's fine to have them on the same page, if that's the case. The register window pops up when clicked and covers the login form, after which the user can close out the window and login after an account has been created.

The issue was that the register.js file was taking the input of "username" and "password" from the login form because the id's for the password and username input fields were the same for both forms. Changing the id for one fixed the issue.

Reply:
Alright, let's leave it as is then.